spec/system/casa_cases/new_spec.rb

```diff
@@ -74,6 +74,23 @@
     end
   end
 
+  context "when the court date field is not filled" do
+    it "does not create a new case" do
+      fill_in "Case number", with: case_number
+      five_years = (Date.today.year - 5).to_s
+      select "March", from: "casa_case_birth_month_year_youth_2i"
+      select five_years, from: "casa_case_birth_month_year_youth_1i"
+
+      within ".actions" do
+        click_on "Create CASA Case"
+      end
+
+      expect(page).to have_current_path(casa_cases_path, ignore_query: true)
+      expect(page).to have_content("Court dates date can't be blank")
+      expect(page.find_field("Next Court Date")[:required]).to eq("required")
```

Comment:
Do you think it can follow the `Single expectation test`? (https://www.betterspecs.org/)

Reply:
In general system specs should not be single-expectation tests because they are more integration tests and test many things at once and are "heavy" (slow) to run. Unit tests should be much more `Single expectation test`

Reply:
Great, @compwron.